pyperlib/drawing/color.py

```diff
@@ -36,7 +49,32 @@ def from_user_input(cls, user_input):
                 return cls(r, g, b, a)
         
         if type(user_input) is str:
+            if user_input in _COLORS_CACHE.keys():
```

Comment:
My bad xD If we’re trying to be efficient e should probably just check if key in dict: